### PR TITLE
[PM-2813] Config service changes

### PR DESCRIPTION
**Pull request by Hinton**
## Type of change

```
- [ ] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

Prevent ConfigService from subscribing multiple times both in popup and background. It also ensures the popup and backgrounds stays in sync.

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team
- Ensure that all UI additions follow [WCAG AA requirements](https://contributing.bitwarden.com/contributing/accessibility/)

### CODE REVIEW

**Review by MGibson1 — COMMENTED** [review]
It looks like this _should_ work, but it depends on the browser sync decorators to do _anything._ Those haven't been terribly reliable, have you had much trouble with it during your testing?

**Comment by Hinton** [comment]
It seemed to work during my testing. I did have some issues with dead objects but it worked fine last I tested on firefox with various configs of open sidebar and popups.

**Review by MGibson1 — APPROVED** [review]

File: libs/common/src/platform/services/config/config.service.ts [hunk 1]
```diff
@@ -18,18 +18,21 @@ export class ConfigService implements ConfigServiceAbstraction {
...
   ) {
-    // Re-fetch the server config every hour
-    timer(0, 1000 * 3600)
-      .pipe(concatMap(() => from(this.fetchServerConfig())))
-      .subscribe((serverConfig) => {
-        this._serverConfig.next(serverConfig);
-      });
+    if (subscribe) {
+      // Re-fetch the server config every hour
+      timer(0, 1000 * 3600)
```
Comment (withinfocus) [inline libs/common/src/platform/services/config/config.service.ts:26]: ⛏️ Not your change but would be nice for this to be a constant somewhere.

Comment (eliykat) [reply libs/common/src/platform/services/config/config.service.ts:26]: I've done this in my PR, https://github.com/bitwarden/clients/pull/5602, which will follow this one.

**Comment by eliykat** [comment]
@Hinton I incorporated these changes into #5602 which should be Ready for QA shortly. I suggest we close this PR as no longer required, and QA can just test your changes as part of mine. But let me know what you want to do.

**Comment by eliykat** [comment]
Closed since #5602 has been merged which included these changes.

**Pull request closed without merging**